modules/ROOT/pages/audit-logging.adoc

```diff
@@ -11,7 +11,9 @@ Audit logs have a retention period of six years. Download your logs periodically
 
 == Access Audit Logging
 
-Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on the Anypoint Platform have access to both the UI and the Query API. The audit log service is business group aware, meaning, you will see only logs that are relevant to your own business group. The audit log UI is embedded in Access Manager. If you are not an *Organization Administrator*, but you have the *Audit Log Viewer* role, then you can see only the audit log user interface under Access Manager.
+Users belonging to the *Organization Administration* role or the *Audit Log Viewer* role on the Anypoint Platform have access to both the UI and the Query API. The audit log service is business group aware, which means you see only logs that are relevant to your own business group. 
```

Comment:
no "the" before Anypoint Platform

Reply:
And should that be business-group aware, since "business group" modifies "aware"?